antarest/core/cache/main.py

```diff
@@ -0,0 +1,24 @@
+from antarest.core.cache.business.local_chache import LocalCache
+from antarest.core.cache.business.redis_cache import RedisCache
+from antarest.core.config import Config
+from antarest.core.interfaces.cache import ICache
+
+
+def build_cache(
+    config: Config,
+) -> ICache:
+    cache_conf = config.cache
+    redis_conf = config.cache.redis
+    cache = (
+        RedisCache(config=redis_conf)
+        if redis_conf is not None
```

Comment:
here we can pass the redis optional client instance instead (that can be None, and will be also used by event_bus)